src/components/OSCALEditableTextField.test.js

```diff
@@ -74,11 +75,39 @@ export default function testOSCALEditableTextField(
       "textField-system-security-plan-metadata-title"
     );
 
-    fireEvent.keyDown(textField, {
+    fireEvent.keyPress(textField, {
       key: "Enter",
       charCode: 13,
     });
 
-    expect(textField).not.toBeVisible();
+    const textFieldAfterEvent = screen.getByTestId(
+      "textField-system-security-plan-metadata-title"
+    );
+
+    expect(textFieldAfterEvent.value).toEqual(textField.value);
+  });
+
+  test(`${parentElementName} shows value saves changes on edit (Enter)`, () => {
+    renderer();
+
+    screen
+      .getByRole("button", {
+        name: "edit-system-security-plan-metadata-title",
+      })
+      .click();
+
+    const textField = screen.getByTestId(
+      "textField-system-security-plan-metadata-title"
+    );
+    fireEvent.change(textField, { target: { value: "New Test Title" } });
+    fireEvent.keyPress(textField, {
+      key: "Enter",
+      charCode: 13,
```

Comment:
Is there a particular reason why this is `charCode` but escape is `keyCode`?

Reply:
There was no specific reason, I changed it for the sake of consistency, Great catch Kyle!